db/sqitch.plan

```diff
@@ -32,3 +32,4 @@ mutations/create_application 2022-08-31T16:16:23Z Anthony Bushara <anthony@butto
 @1.0.1 2022-09-02T00:40:17Z Matthieu Foucault <matthieu@button.is> # release v1.0.1
 @1.1.0 2022-09-06T23:59:22Z Matthieu Foucault <matthieu@button.is> # release v1.1.0
 @1.1.1 2022-09-09T20:35:47Z Marcel Mueller,,, <marcel@Thinkpad> # release v1.1.1
+mutations/submit_application [mutations/submit_application@1.0.0] 2022-09-13T19:52:43Z Marcel Mueller,,, <marcel@button.is> # Check all submission fields in form_data are filled before submission
```

Comment:
```suggestion
mutations/submit_application [mutations/submit_application@1.0.0] 2022-09-13T19:52:43Z Marcel Mueller <marcel@button.is> # Check all submission fields in form_data are filled before submission
```

You should configure sqitch so that the correct user name is added in the plan